website/markdown/posts/2020-02-26-version-1.3.0/post.mdx

```diff
@@ -0,0 +1,90 @@
+As soon as users reported, that caused some issues because GitHub's API returned limit errors.
+To mitigate this issue,
+we moved our artifacts to Google Cloud Storage.
+Moreover,
+we took the opportunity to add support for pulling binaries for incremental builds.
+For instance,
+if a developer wants to try out a specific commit,
+they can run `tuist local commit_sha` and Tuist will pull the binaries and make sure that we are running the right version.
+
+## Other fixes and improvements
+
+Tuist 1.3.0 also ships with minor fixes and improvements that are worth mentioning:
+
+- Added metal as a valid source extension - [Link](https://github.com/tuist/tuist/pull/1023)
+- Fixed false positive warnings during the linting process - [Link](https://github.com/tuist/tuist/pull/981)
+- Added support for target action paths without the `./` prefix - [Link](https://github.com/tuist/tuist/pull/997)
+- Ensured bcsymbolmap paths are consistently sorted
+- Updated XcodeProj to 7.8.0
+- Added "Base" to the list of known regions - [Link](https://github.com/tuist/tuist/pull/1021)
+
+## What’s next
+
+Our goal is to provide teams with a full-fledged tool that helps them deal with their projects at scale. The list of challenges that teams face when scaling up their projects is long:
+
+- Keep consistency across projects.
+- Ensure that projects are simple and that can be modularized easily.
+- Integrate the management of certificates into.
+- Have insights about build times and the quality of the project.
+- Slow build times.
+
+We are starting to take the steps towards helping teams with those challenges. In fact, we are implementing a simple and standard interface to build and test projects. With it, projects no longer need another layer _(e.g. Fastfiles)_ that can become complex and hard to reason about. If you are in a directory where there’s a Project.swift or a Workspace.swift, you’ll be able to run `tuist build` and `tuist test`.
+
+Moreover, we are implementing caching by leveraging [xcframeworks](https://developer.apple.com/videos/play/wwdc2019/416/) and project generation. Teams might need to simplify their projects and remove side effects, for example, introduced by script build phases, but the benefits are huge. The only source code that developers will need to compile is the code of the module that they are working on. Transitive and non-transitive dependencies will be xcframeworks.
+
+> In other words, we are making scaling Xcode projects easy and fun for the rest of us
+
+It’s an ambitious plan but we are confident we can get there. We can’t move as fast as we’d like to but this slow yet with a steady pace is helping us keep an eye on the architecture and ensuring it evolves sustainably with the project. If you’d like to be part of the journey and contribute to making it happen join our [Slack](https://slack.tuist.io) and let us know. We’ll gladly onboard you and give you pointers to start contributing.
```

Comment:
I'd delete `with a` (ie `this slow yet steady pace`)

Reply:
Good idea. Changed ✅